tasks/auth_initialization.yml

```diff
@@ -9,7 +9,7 @@
     name: "{{ item.name }}"
     password: "{{ item.password }}"
     roles: "{{ item.roles }}"
-    login_host: 127.0.0.1
+    login_host: "{{ mongodb_listen_iface }}"
```

Comment:
I found out it. But, I think it's not iface, it's an ip address. Why you called it iface?